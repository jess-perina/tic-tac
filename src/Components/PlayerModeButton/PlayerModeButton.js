import React from 'react';

function PlayerModeButton(props) {
  return (
    <div className="player-mode mode-btn">
      <span className="card-title">Change Player Mode</span>
      <a className="dropdown-trigger btn" href="#" data-target="player-mode">{props.player}</a>
      <ul id="player-mode" className="dropdown-content">
        <li onClick={props.setSinglePlayer}>
          <a href="#!">Single Player</a>
        </li>
        <li onClick={props.setTwoPlayer}>
          <a href="#!">Two Player</a>
        </li>
      </ul>
    </div>
  )
}

export default PlayerModeButton
