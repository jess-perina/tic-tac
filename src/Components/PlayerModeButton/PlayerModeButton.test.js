import React from 'react'
import { shallow } from 'enzyme'
import PlayerModeButton from './PlayerModeButton'

describe('PlayerModeButton Component', () => {
  it('renders as espected', () => {
    expect(shallow(<PlayerModeButton />)).toMatchSnapshot()
  })
})