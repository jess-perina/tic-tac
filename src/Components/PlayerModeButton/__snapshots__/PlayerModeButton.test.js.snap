// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerModeButton Component renders as espected 1`] = `ShallowWrapper {}`;
